src/lib/gundb/UserStorage.js

```diff
@@ -363,12 +379,77 @@ class UserStorage {
         })
     })
     let results = flatten(await Promise.all(promises))
+    logger.debug({ results, daysToTake, cursor: this.cursor })
+    // const stdResults = results.map(this.standardizeFeed)
+    // console.log('stdResults', stdResults)
     return results
   }
+
+  async getStandardizedFeed(amount: number, reset: boolean): Promise<Array<StandardFeed>> {
+    const feed = await this.getAllFeed()
+    logger.info({ feed })
+    gun.get('users').load(allUsers => logger.info({ allUsers }), { wait: 99 })
+
+    return await Promise.all(feed.filter(feedItem => feedItem.data).map(this.standardizeFeed))
+    // TODO: Use proper pagination
+    // return (await this.getFeedPage(amount, true)).map(this.standardizeFeed)
+  }
+
+  async standardizeFeed(feedEvent: FeedEvent): Promise<StandardFeed> {
+    const { data } = feedEvent
+    const { receipt } = data
```

Comment:
It would be nice to extract a method for getting for an event. So this standarizeFeed looks claner.